### PR TITLE
Simplifying query commands

### LINKED ISSUE #378: Remove unnecessary query commands
- Remove `query compute contract-state raw` & `query compute contract-state all`
- Change the name of `query compute contract-state smart` to something simpler

### PR DESCRIPTION
**Pull request by toml01**
closes #378

### CODE REVIEW

**Comment by Cashmaney** [comment]
Also can we comment out x/compute/client/cli/new_tx.go?

**Comment by toml01** [comment]
Sorry, forgot to mention `WIP`

File: cosmwasm/packages/wasmi-runtime/src/wasm/contract_operations.rs [hunk 1]
```diff
@@ -201,13 +201,13 @@ pub fn query(
...
     debug!(
-        "Query input before encryption: {:?}",
+        "Query input before decryption: {:?}",
         String::from_utf8_lossy(&msg)
     );
     let secret_msg = SecretMessage::from_slice(msg)?;
     let decrypted_msg = secret_msg.decrypt()?;
     debug!(
-        "Query input afer encryption: {:?}",
+        "Query input afer decryption: {:?}",
```
Comment (assafmo) [inline cosmwasm/packages/wasmi-runtime/src/wasm/contract_operations.rs:210]: Thank. Could you also fix this in handle & init?

Comment (toml01) [reply cosmwasm/packages/wasmi-runtime/src/wasm/contract_operations.rs:210]: Didn't notice, I will

**Pull request merged**